### PR TITLE
[RA1][RC1] Dump RC1 in RA1 Chapter5

### PR DESCRIPTION
**Pull request by collivier**
Targeting the next GSMA RA1 updates, it extracts a part of:
- RC/index.rst
- RC1/chapters/chapter03
- RC1/chapters/chapter04

Signed-off-by: Cédric Ollivier <cedric.ollivier@orange.com>

### CODE REVIEW

**Comment by collivier** [comment]
@pgoyal01 @karinesevilla this is an early overview about the content we could dump.
it would remain to fix a couple of obsolete terms and possibly to add new content.

**Review by pgoyal01 — APPROVED** [review]

**Comment by pgoyal01** [comment]
@collivier do we have reference to Chapter 8 in any of the other RA1 chapters that would need to be changed?

**Comment by karinesevilla** [comment]
@collivier I globally agree on this new content for chapter 8. We will need to work on the text to remove what is related to Kubernetes testing, RC1, and RC2.

**Comment by collivier** [comment]
@karinesevilla the funtest kubernetes is in Functest in a nutshell. but yes we could remove this part in next PR

**Review by karinesevilla — APPROVED** [review]

**Review by ildikov — COMMENTED** [review]
I know that we agreed to iterate on the content in follow up PRs. However, there seem to be a lot of content here that I'm not sure should be in RA1. Is all this content moved over from RC1 or will this be covered there too?

Thanks,
Ildikó

**Comment by collivier** [comment]
@ildikov no. As we discussed last RA1's meeting, we will remove RC1 after the merge. There will be no need to keep an obsolete and duplicated tree. Could we merge as we all agreed yesterday to push the new PR on top of it. Thank you in advance.

File: doc/ref_arch/openstack/chapters/chapter08.rst [hunk 1]
```diff
@@ -1,35 +1,1698 @@
...
+
+The verification, validation, and conformance processes leverage
+existing Anuket testing knowledge (projects) and experience (history) by
+utilising the toolchain design already in-place. The conformance
+toolchain only requires for the local deployment of the components instead of
+leveraging the common Anuket centralized services. However, the
+interfaces remain unchanged for leveraging test jobs, the common test
+case execution, the test result database and the S3 protocol to publish
+the artifacts. It’s worth mentioning that dumping all results and logs
+required for conformance is already in place in Functest daily jobs (see
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter08.rst:100]: should "functest" be listed in the Terminology?

File: doc/ref_arch/openstack/chapters/chapter08.rst [hunk 2]
```diff
@@ -1,35 +1,1698 @@
...
+`functest-wallaby-zip <https://build.opnfv.org/ci/job/functest-wallaby-zip/4/console>`__).
+
+It should be noted that `Xtesting
+CI <https://galaxy.ansible.com/collivier/xtesting>`__ supports both
+centralized and distributed deployment models as described before. It
+has deployed the full toolchain in one small virtual machine to verify
+ONAP Openlab via Functest.
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter08.rst:107]: should we add ONAP and OpenLab to the Terminology?

File: doc/ref_arch/openstack/chapters/chapter08.rst [hunk 3]
```diff
@@ -1,35 +1,1698 @@
...
+   and requirements to demonstrate traceability and coverage.
+
+Test Case Integration and Tooling
+---------------------------------
+
+The OpenStack based cloud infrastructure suite must utilize the Anuket test
+case integration toolchain to deliver overall integration, the same end user
+actions, and a unique test result format (e.g. Anuket test result
+database) needed by the end users and any test case result verification
+program (e.g. `OVP <https://www.opnfv.org/verification>`__).
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter08.rst:76]: Should we add OVP to Terminology?

File: doc/ref_arch/openstack/chapters/chapter08.rst [hunk 4]
```diff
@@ -1,35 +1,1698 @@
...
+
+Anuket Toolchains
+~~~~~~~~~~~~~~~~~
+
+Anuket, previously named OPNFV, has built a complete CI/CD toolchain for
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter08.rst:81]: Should we add Anuket, OPNFV, and CI/CD to Terminology?

File: doc/ref_arch/openstack/chapters/chapter08.rst [hunk 5]
```diff
@@ -1,35 +1,1698 @@
...
+----------------------
+
+Functest in a nutshell
+~~~~~~~~~~~~~~~~~~~~~~
+
+`Functest <https://functest.readthedocs.io/en/stable-xena/>`__ was
+initially created to verify OPNFV Installers and Scenarios and then to
+publish fair, trustable and public results regarding the status of the
+different opensource technologies, especially for Neutron backends
+(e.g. Neutron agents, OpenDaylight, OVN, etc.). It has been continuously
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter08.rst:194]:
```suggestion
(e.g., Neutron agents, OpenDaylight, OVN, etc.). It has been continuously
```

File: doc/ref_arch/openstack/chapters/chapter08.rst [hunk 6]
```diff
@@ -1,35 +1,1698 @@
...
+Functest is driven by a true verification of the platform under test as
+opposed to the interoperability programs such as
+`RefStack <https://refstack.openstack.org/>`__ or `OPNFV Verification
+Program <https://www.opnfv.org/verification>`__ which select a small
+subset of Functional tests passing in many different opensource software
+combinations:
+
+-  tests are skipped if an optional support is missing (e.g.
+   `Barbican <https://docs.openstack.org/barbican/latest/>`__ or
+   networking features such as `BGPVPN
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter08.rst:209]: Should we add BGPVPN to Terminology?

File: doc/ref_arch/openstack/chapters/chapter08.rst [hunk 7]
```diff
@@ -1,35 +1,1698 @@
...
+
+It should be noted that `the RefStack
+lists <https://refstack.openstack.org/>`__ are included as
+they are in Functest in the next 3 dedicated testcases:
+
+-  refstack_compute (OpenStack Powered Compute)
+-  refstack_object (OpenStack Powered Object Storage)
+-  refstack_platform (OpenStack Powered Platform)
+
+Functest also integrates `Kubernetes End-to-end
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter08.rst:224]: Should we add Kubernetes to Terminology?

File: doc/ref_arch/openstack/chapters/chapter08.rst [hunk 8]
```diff
@@ -1,35 +1,1698 @@
...
+tests <https://kubernetes.io/blog/2019/03/22/kubernetes-end-to-end-testing-for-everyone/>`__
+and allows verifying Kubernetes Conformance (see
+`k8s-conformance <https://build.opnfv.org/ci/job/functest-kubernetes-opnfv-functest-kubernetes-smoke-v1.22-k8s_conformance-run/25/console>`__).
+
+Then Functest conforms with the upstream rules (versions, code quality,
+etc.) and especially their
+`gates <https://docs.opendev.org/opendev/system-config/latest/devstack-gate.html>`__
+(a.k.a. the automatic verification prior to any code review) to preserve
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter08.rst:232]: Should we add "a.k..a" and "aka" to Terminology?

File: doc/ref_arch/openstack/chapters/chapter08.rst [hunk 9]
```diff
@@ -1,35 +1,1698 @@
...
+And VNFs automatically deployed and tested :
+
+.. table:: VNFs
+   :widths: auto
+
+   ============ ===================================
+   Testcases    Benchmarking
+   ============ ===================================
+   cloudify     Cloudify deployment
+   cloudify_ims Clearwater IMS deployed via Coudify
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter08.rst:296]: Should we add IMS to Terminology?

File: doc/ref_arch/openstack/chapters/chapter08.rst [hunk 10]
```diff
@@ -1,35 +1,1698 @@
...
+   heat_ims     Clearwater IMS deployed via Heat
+   vyos_vrouter VyOS deployed via Cloudify
+   juju_epc     OAI deployed via Juju
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter08.rst:299]: Should we add OAI to Terminology?

File: doc/ref_arch/openstack/chapters/chapter08.rst [hunk 11]
```diff
@@ -1,35 +1,1698 @@
...
+successfully.
+
+New `Functest
+containers <https://lists.opnfv.org/g/opnfv-tsc/message/5717>`__ have
+been proposed for Anuket Compliance which simply override the default
+test configurations and the default test lists. Any optional capability
+or services (e.g. Barbican) can be still verified by the classical
+Functest containers.
+
+The next subsections only detail the Tempest tests which must not be
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter08.rst:352]:
```suggestion
The next subsections detail the Tempest tests which must not be
```

File: doc/ref_arch/openstack/chapters/chapter08.rst [hunk 12]
```diff
@@ -1,35 +1,1698 @@
...
+executed from a Compliance state point. The remaining tests have to pass
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter08.rst:353]:
```suggestion
executed from a compliance state point. The remaining tests have to pass
```

File: doc/ref_arch/openstack/chapters/chapter08.rst [hunk 13]
```diff
@@ -1,35 +1,1698 @@
...
+-  NeutronNetworks.set_and_clear_router_gateway
+-  Quotas.neutron_update
+
+Compute - Nova API testing
+''''''''''''''''''''''''''
+
+Nova API is covered in the OpenStack Gates via
+`Tempest <https://opendev.org/openstack/tempest>`__ as integrated in
+`Functest Smoke
+CNTT <https://git.opnfv.org/functest/tree/docker/smoke-cntt/testcases.yaml?h=stable%2Fwallaby>`__.
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter08.rst:805]: CNTT or Anuket

File: doc/ref_arch/openstack/chapters/chapter08.rst [hunk 14]
```diff
@@ -1,35 +1,1698 @@
...
+   ============================================= ==========
+
+Dataplane benchmarking
+^^^^^^^^^^^^^^^^^^^^^^
+
+`Functest Benchmarking
+CNTT <https://git.opnfv.org/functest/tree/docker/benchmarking-cntt/testcases.yaml?h=stable%2Fwallaby>`__
+offers two benchmarking dataplane test cases leveraging on:
+
+-  `VMTP <http://vmtp.readthedocs.io/en/latest>`__
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter08.rst:1420]: Should we add VMTP to Terminology? also I think I missed DHCP? If using CNTT then it should also be added to Terminology

File: doc/ref_arch/openstack/chapters/chapter08.rst [hunk 15]
```diff
@@ -1,35 +1,1698 @@
...
+   ICMP     64       rtt_stddev
+   ICMP     391      rtt_avg_ms
+   ICMP     391      rtt_max_ms
+   ICMP     391      rtt_min_ms
+   ICMP     391      rtt_stddev
+   ICMP     1500     rtt_avg_ms
+   ICMP     1500     rtt_max_ms
+   ICMP     1500     rtt_min_ms
+   ICMP     1500     rtt_stddev
+   UDP      128      loss_rate
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter08.rst:1485]: Should we add UDP to Terminology?

**Comment by pgoyal01** [comment]
I think we are using "SG" in the chapter to mean "Security Group" -- should it be added to abbreviations

**Review by ildikov — APPROVED** [review]

**Pull request merged**